src/raster/r.hand/r.hand.py

```diff
@@ -0,0 +1,416 @@
+#!/usr/bin/env python3
+
+############################################################################
+#
+# MODULE:       r.hand
+# AUTHOR:       Corey T. White, OpenPlains Inc.
+# PURPOSE:      Performs Height Above Nearest Drainage (HAND) analysis.
+# COPYRIGHT:    (C) 2025 OpenPlains Inc.
+#               This program is free software under the GNU General
+#               Public License (>=v2). Read the file COPYING that
+#               comes with GRASS for details.
+#
+#############################################################################
+
+#!/usr/bin/env python
+
+# %module
+# % description: Performs Height Above Nearest Drainage (HAND) analysis.
```

Comment:
It computes the inundation based on HAND, not just HAND, so I would clarify that.